lib/App/Yath/Command/collector.pm

```diff
@@ -28,7 +35,7 @@ sub run {
 
     my $fh = isolate_stdout();
 
-    my $settings = Test2::Harness::Settings->new(File::Spec->catfile($dir, 'settings.json'));
+    my $settings = $self->settings;
```

Comment:
I don't know the effect of removing settings.json here. Is this the right way to do it?

Reply:
See my new commit

Reply:
yep.